docroot/profiles/humsci/su_humsci_profile/config/sync

```diff
@@ -0,0 +1 @@
+../../../../../config/default/
```

Comment:
Symlinks have been problematic for Ian Monroe while on his windows machine. I'll add a topic in Dev chat to see what we can do to keep his work manageable.

Reply:
hmm. thats a bit of an issue

Reply:
we have more symlinks than this thats for sure

Reply:
Yeah, this shouldn't block anything and it appears to be a combination of running composer commands through windows and then running a unix subsystem or docker container to run the code. That mix and match no worky.